wiki/Tournaments/MWC/2014/en.md

```diff
@@ -313,30 +326,30 @@ The osu!mania World Cup 2014 is run by various community members by distributing
   - [Sharlo - Sleeping Butterfly (Rumia-) \[Lv.32\]](https://osu.ppy.sh/beatmaps/392533)
   - [Lite Show Magic - Crack traxxxx (LeiN-) \[7K INF\]](https://osu.ppy.sh/beatmaps/315723)
 - Tiebreaker
-  - [FamiRockP - Oni Kanojo (Entozer) \[Insane\]](https://osu.ppy.sh/beatmaps/192564)
+  - **[FamiRockP - Oni Kanojo (Entozer) \[Insane\]](https://osu.ppy.sh/beatmaps/192564)**
 
 ------------------------------------------------------------------------
 
-## Results: 4K
+## Match Results: 4K
 
 ### Finals
 
 | Team A | | | Team B | History |
```

Comment:
change headers in every results into:
`| Day, date | | | | |`

e.g. `| Sunday, 2018-12-20 | | | | |`